package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "oma-bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.1",
+    "react-scripts": "5.0.0",
+    "web-vitals": "^2.1.3"
+  },
```

Comment:
- [ ] Please install `react-redux` dependency by running `npm install react-redux`. `react-redux` will be needed to use Redux in the next milestones